pglet/grid.py

```diff
@@ -20,22 +38,10 @@ def __init__(self, id=None, name=None, icon=None, icon_only=None,
         self.max_width = max_width
         self.on_click = on_click
 
-        self.__template_controls = []
-        if template_controls != None:
-            for control in template_controls:
-                self.__template_controls.append(control)
+        self.template_controls = template_controls or []
 
     def _get_control_name(self):
         return "column"
-    
```

Comment:
Did you accidentially remove `template_controls` property?

Reply:
Ah, never mind, it's `self.template_controls` now. It's my habbit coming from C# to wrap everything into properties :)